src/lib/Libifl/tm.c

```diff
@@ -1172,6 +1186,8 @@ tm_finalize()
 	init_done = 0;
 	free(tm_jobid);
 	free(tm_jobcookie);
+	tm_jobcookie = NULL;
```

Comment:
I think it will good also if we add `tm_jobid = NULL;` and `tm_jobid_len = 0;` here.

Reply:
Agreed. Done.